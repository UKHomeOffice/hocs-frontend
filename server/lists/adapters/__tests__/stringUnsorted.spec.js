const stringUnsortedAdapter = require('../stringUnsorted');

const mockLogger = {
    debug: () => { },
    info: () => { },
    warn: () => { },
    error: () => { }
};

describe('String Unsorted Adapter', () => {

    it('should transform but not sort string data', async () => {
        const mockData = [
            'String B',
            'String C',
            'String A'
        ];

        const results = await stringUnsortedAdapter(mockData, { logger: mockLogger });

        expect(results).toBeDefined();
        expect(results).toMatchSnapshot();
    });
});