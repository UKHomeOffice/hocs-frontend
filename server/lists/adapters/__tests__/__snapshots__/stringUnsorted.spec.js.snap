// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`String Unsorted Adapter should transform but not sort string data 1`] = `
Array [
  "String B",
  "String C",
  "String A",
]
`;
